test/limits/mzcompose.py

```diff
@@ -2006,15 +2006,57 @@ def workflow_main(c: Composition, parser: WorkflowArgumentParser) -> None:
     if not scenarios:
         return
 
-    setup(c, args.workers)
+    with c.override(
+        Clusterd(
+            name="clusterd_1_1_1",
+            workers=args.workers,
+            process_names=["clusterd_1_1_1", "clusterd_1_1_2"],
+        ),
+        Clusterd(
+            name="clusterd_1_1_2",
+            workers=args.workers,
+            process_names=["clusterd_1_1_1", "clusterd_1_1_2"],
+        ),
+        Clusterd(
+            name="clusterd_1_2_1",
+            workers=args.workers,
+            process_names=["clusterd_1_2_1", "clusterd_1_2_2"],
+        ),
+        Clusterd(
+            name="clusterd_1_2_2",
+            workers=args.workers,
+            process_names=["clusterd_1_2_1", "clusterd_1_2_2"],
+        ),
+        Clusterd(
+            name="clusterd_2_1_1",
+            workers=args.workers,
+            process_names=["clusterd_2_1_1", "clusterd_2_1_2"],
+        ),
+        Clusterd(
+            name="clusterd_2_1_2",
+            workers=args.workers,
+            process_names=["clusterd_2_1_1", "clusterd_2_1_2"],
+        ),
+        Clusterd(
+            name="clusterd_3_1_1",
+            workers=1,
+        ),
+    ):
+        run_scenarios(c, scenarios, args.find_limit, args.workers)
```

Comment:
I mainly added the `run_scenarios` method so I don't have to indent all subsequent code by one. I think it helps readability but you might disagree!